cmd/up.go

```diff
@@ -152,67 +156,36 @@ func (up *UpContext) Activate() {
 			up.Exit <- err
 			return
 		}
-
 		fmt.Println(" ✓  Files synchronized")
 
-		progress = newProgressBar("Finalizing configuration...")
-		progress.start()
-		err = up.forceLocalSyncState()
-		progress.stop()
-		if err != nil {
-			up.Exit <- err
-			return
-		}
-
-		switch prevError {
-		case errors.ErrLostConnection:
-			log.Green("Reconnected to your cluster.")
-		}
-
 		printDisplayContext("Your Okteto Environment is ready", up.Dev.Namespace, up.Dev.Name, up.Dev.Forward)
-		cmd, port := up.buildExecCommand()
-		if err := cmd.Start(); err != nil {
-			log.Infof("Failed to execute okteto exec: %s", err)
-			up.Exit <- err
-			return
-		}
-
-		log.Debugf("started new okteto exec")
 
 		go func() {
-			up.WG.Add(1)
```

Comment:
No need to wait for the exec, this way we do not need to create a new process.
- If the command exists, it is notified via the `Running` channel.
- If not:
  a) the `okteto up` finishes due to connectivity issues and the `exec` is killed.
  b) `okteto up` retries and it will kill the exec with `kill -- -1`